src/Slider.js

```diff
@@ -151,6 +183,7 @@ var Slider = React.createClass({
       thumbTintColor: '#343434',
       thumbTouchSize: {width: 40, height: 40},
       debugTouchArea: false,
+      animationType: 'spring'
```

Comment:
Could you set the default to `timing` instead?
I feel that's what most people would expect.